CMakeLists.txt

```diff
@@ -39,6 +34,10 @@ set(SHORT_NAME julius)
 set(USER_FRIENDLY_NAME Julius)
 project(${SHORT_NAME} C)
 
+if (NINTENDO_SWITCH)
```

Comment:
Where does this `NINTENDO_SWITCH` come from?

Reply:
This comes from our cmake platform configuration https://github.com/devkitPro/pacman-packages/blob/master/cmake/switch/NintendoSwitch.cmake#L11